plextraktsync/plex_api.py

```diff
@@ -629,10 +629,16 @@ def update_playlist(self, name: str, items: List[Union[Movie, Show, Episode]]) -
         """
         Updates playlist (creates if name missing) replacing contents with items[]
         """
+        playlist: Playlist = None
         try:
             playlist: Playlist = self.plex.playlist(name)
```

Comment:
this. type hint is now superfluous

Reply:
Would this fix it? Or is type hinting not needed when assigning None? 

```
playlist: Playlist = None
try:
    playlist = self.plex.playlist(name)
except NotFound:
    if len(items) > 0:
        playlist = self.plex.createPlaylist(name, None, items)
```

Reply:
yep. that would do.